index.html

```diff
@@ -0,0 +1,138 @@
+<html>
+
+<head>
+  <title>Bootstrap 4 Layout</title>
+  <meta http-equiv="x-ua-compatible" content="ie=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Raleway:400,800">
+  <link rel='stylesheet' href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+  <!--  <link rel="stylesheet" href="src/css/styles.css">
+    <link rel="stylesheet" href="src/css/styles.css">
+    <link rel="stylesheet" href="src/multirange.css">-->
+</head>
+
+<body>
+  <script src="dist/bundle.js"></script>
```

Comment:
Try and move this in the footer. See if it still works. 

it's better to have the JS loaded after the DOM rendered. Better performance